### ISSUE TITLE
Task list is not unique

### ISSUE THREAD

**Issue by kc97ble** [CS2103-AY1819S1-W17-4/main#195]
Input 1

```
add n/CS2103T Peer Feedback p/1 f/7 d/10/11/2018 t/cs2103t t/project
add n/CS2103T Peer Feedback p/1 f/7 d/10/11/2018 t/cs2103t t/project
```

Output 1

```
New task added: CS2103T Peer Feedback Priority: 1 Frequency: 7 Deadline: 10/11/2018 Tags: [project][cs2103t] Attachments: 
This task already exists in the deadline manager
```

Input 2

```
add n/CS2103T Peer Feedback p/1 f/7 d/10/11/2018 t/cs2103t t/project
add n/CS2103T Peer Feedback p/2 f/7 d/10/11/2018 t/cs2103t t/project
edit 2 p/1
```

Output 2

```
New task added: CS2103T Peer Feedback Priority: 1 Frequency: 7 Deadline: 10/11/2018 Tags: [project][cs2103t] Attachments: 
New task added: CS2103T Peer Feedback Priority: 2 Frequency: 7 Deadline: 10/11/2018 Tags: [project][cs2103t] Attachments: 
Edited Task: CS2103T Peer Feedback Priority: 1 Frequency: 7 Deadline: 10/11/2018 Tags: [project][cs2103t] Attachments: 
```

![image](https://user-images.githubusercontent.com/7184368/47717587-24238680-dc81-11e8-8e01-a37a0aa22a3b.png)

**Comment by ranaldmiao** [comment]
I think that this unique check constraint is only in force for 'add command'.

I suggest we allow duplicated tasks and remove the check for add command.

**Comment by kc97ble** [comment]
> I suggest a way to solve this problem: we add a timestamp, which is completely hidden to the user, by doing so, all the tasks will be unique, and import/export feature will still work

By doing this, the app will still work properly even when we import twice.

**Comment by sijie123** [comment]
Importing twice already works :O

**Comment by ranaldmiao** [comment]
Yes, so only 'add' command is the issue now.

**Comment by ranaldmiao** [comment]
Flagged by #245 and #250 as well.

**Comment by ranaldmiao** [comment]
Fixed by #286.
